include/spdlog/sinks/stdout_sinks-inl.h

```diff
@@ -15,7 +15,11 @@
 // under windows using fwrite to non-binary stream results in \r\r\n (see issue #1675)
 // so instead we use ::FileWrite
 #include <spdlog/details/windows_include.h>
+
+#ifndef _USING_V110_SDK71_
```

Comment:
Please add comment about the purpose of this ifdef. It woild hard to understand without it.

Reply:
The **_USING_V110_SDK71_** guard works when building based on a pre-8 version of Windows SDK, or trying to use the XP-toolset like **vc141_xp**.